package.json

```diff
@@ -0,0 +1,87 @@
+{
+  "name": "@openmrs/esm-htmlforms",
+  "version": "1.0.0",
+  "description": "HTML Form Entry forms embedded in OpenMRS SPA",
+  "main": "dist/openmrs-esm-htmlforms.js",
+  "scripts": {
+    "start": "webpack-dev-server",
+    "start-https": "webpack-dev-server --https",
+    "build": "webpack --mode production",
+    "lint": "eslint src --ext tsx",
+    "prettier": "prettier",
+    "typescript": "tsc",
+    "prepublishOnly": "npm run build",
+    "test": "jest --config jest.config.json"
+  },
+  "directories": {
+    "lib": "dist"
+  },
+  "browserslist": [
+    "extends browserslist-config-openmrs"
+  ],
+  "husky": {
+    "hooks": {
+      "pre-commit": "pretty-quick --staged && concurrently 'npm:lint' 'npm:test' 'npm:typescript'"
+    }
+  },
+  "types": "src/openmrs-esm-htmlforms.tsx",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/openmrs/openmrs-esm-htmlforms.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/openmrs/openmrs-esm-htmlforms/issues"
+  },
+  "homepage": "https://github.com/openmrs/openmrs-esm-htmlforms#readme",
+  "devDependencies": {
+    "@babel/core": "^7.5.5",
+    "@babel/plugin-proposal-class-properties": "^7.5.5",
+    "@babel/preset-env": "^7.5.5",
+    "@babel/preset-react": "^7.0.0",
+    "@babel/preset-typescript": "^7.3.3",
+    "@openmrs/react-root-decorator": "^1.0.0",
+    "@testing-library/react": "^9.1.3",
+    "@types/jest": "^24.0.18",
+    "babel-eslint": "^10.0.2",
+    "babel-jest": "^24.9.0",
+    "babel-loader": "^8.0.6",
+    "browserslist-config-openmrs": "^1.0.0",
+    "clean-webpack-plugin": "^3.0.0",
+    "concurrently": "^5.0.0",
+    "css-loader": "^3.2.0",
+    "enzyme": "^3.10.0",
+    "enzyme-adapter-react-16": "^1.14.0",
+    "eslint": "^6.2.1",
+    "eslint-config-prettier": "^6.1.0",
+    "eslint-config-ts-react-important-stuff": "^1.0.0",
+    "eslint-plugin-prettier": "^3.1.0",
+    "fork-ts-checker-webpack-plugin": "^1.5.0",
+    "husky": "^3.0.4",
+    "identity-obj-proxy": "^3.0.0",
+    "jest": "^24.9.0",
+    "jest-cli": "^24.9.0",
+    "kremling": "^2.0.1",
+    "prettier": "^1.18.2",
+    "pretty-quick": "^1.11.1",
+    "react-router-dom": "^5.0.1",
+    "single-spa-react": "^2.10.2",
+    "style-loader": "^1.0.0",
+    "systemjs-webpack-interop": "^1.1.0",
+    "typescript": "^3.5.3",
+    "webpack": "^4.39.2",
+    "webpack-cli": "^3.3.7",
+    "webpack-dev-server": "^3.8.0"
+  },
+  "dependencies": {
+    "@openmrs/esm-api": "^1.0.3",
+    "@openmrs/esm-module-config": "^1.0.0",
+    "@types/react": "^16.9.2",
+    "@types/react-dom": "^16.9.0",
+    "@types/react-router-dom": "^4.3.5",
+    "react": "^16.9.0",
+    "react-dom": "^16.9.0",
+    "rxjs": "^6.5.3"
```

Comment:
`rxjs` recently got moved out of the import map, in favor of each package bundling it individually into their packages. See related https://github.com/openmrs/openmrs-esm-root-config/pull/30.

As such, we should move `rxjs` to devDependencies.